text_collectors/virt.sh

```diff
@@ -54,7 +54,7 @@ then
       "$( dmidecode -s bios-vendor )" "$( dmidecode -s bios-version )" \
       "$( dmidecode -s system-manufacturer )" "$( dmidecode -s system-product-name )" "$( dmidecode -s system-version )" \
       "$( dmidecode -s baseboard-manufacturer )" "$( dmidecode -s baseboard-product-name )" )"
-    echo "${line//\\ / }" >>"${v}"
+    echo "${line//\\/ }" >>"${v}"
```

Comment:
can we have a small comment explaining the rationale? and maybe a link to the BZ.

also the BZ target release should be changed to 4.7.0.